packages/common/log.js

```diff
@@ -1,36 +1,103 @@
 'use strict';
+const util = require('util');
 
-const tagged = (log, tag, thisObj) =>
-  log.bind(thisObj || console, `[${tag}]`);
-
-const use = (log, error = null) =>
-  Object.assign({}, module.exports, {
-    log: log,
-    info: tagged(log, 'INFO'),
-    warn: tagged(log, 'WARN'),
-    debug: tagged(log, 'DEBUG'),
-    error: tagged(error || log, 'ERROR')
-  });
-
-const logs = use(console.log, console.error); // eslint-disable-line no-console
-module.exports = Object.assign({}, logs);
-
-const reset = () => {
-  Object.assign(module.exports, logs);
-};
-
-module.exports = {
-  use: use,
-  tagged: tagged,
-  mute: (...levels) => {
-    reset();
-    for (const level of levels) {
-      module.exports[level] = () => null;
-    }
-  },
-  unmute: () => {
-    reset();
-  }
-};
-
-reset();
+/**
+ * Constructs JSON to log
+ *
+ * @param {string} level - type of log (info, error, debug, warn)
+ * @param {string} args - Message to log
+ * @returns {undefined} - log is printed to stdout, nothing is returned
+ */
+function log(level, args, additionalKeys) {
+  const time = new Date();
+  let output = {
+    level,
+    executions: process.env.EXECUTIONS,
+    timestamp: time.toISOString(),
+    sender: process.env.SENDER
+  };
+
+  const message = util.format.apply(null, args);
```

Comment:
Could this be directly assigned to output.message?  I don't see `message` used anywhere else in this function.